
From: Darren Williams <dsw@gelato.unsw.edu.au>

include/asm/checksum.h:75: warning: `struct in6_addr' declared inside parameter list include/asm/checksum.h:75

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/include/asm-alpha/checksum.h |    1 +
 1 files changed, 1 insertion(+)

diff -puN include/asm-alpha/checksum.h~267-fix-broken-alpha-build-ptracec-error include/asm-alpha/checksum.h
--- 25/include/asm-alpha/checksum.h~267-fix-broken-alpha-build-ptracec-error	2004-06-23 23:36:49.437954840 -0700
+++ 25-akpm/include/asm-alpha/checksum.h	2004-06-23 23:36:49.441954232 -0700
@@ -1,6 +1,7 @@
 #ifndef _ALPHA_CHECKSUM_H
 #define _ALPHA_CHECKSUM_H
 
+#include <linux/in6.h>
 
 /*
  *	This is a version of ip_compute_csum() optimized for IP headers,
_
